packages/payload/src/config/types.ts

```diff
@@ -730,6 +730,42 @@ export type AfterErrorHook = (
   args: AfterErrorHookArgs,
 ) => AfterErrorResult | Promise<AfterErrorResult>
 
+export type Widget = {
+  ComponentPath: string
+  /** If undefined, will be set to 3 */
+  maxHeight?: 1 | 2
+  /** If undefined, will be set to 12 */
+  maxWidth?: 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11
+  /** If undefined, will be set to 1 */
+  minHeight?: 2 | 3
+  /** If undefined, will be set to 3 */
+  minWidth?: 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
```

Comment:
Following feedback, I added controls for setting minimum and maximum widget sizes.

Reply:
Yes, I was unsure about this myself. Actually, we're not being entirely consistent right now, and in some places we're excluding default values ​​from types. But I agree that it feels difficult to parse here. I'll listen to you!